Source/santad/DataLayer/WatchItems.mm

```diff
@@ -530,13 +531,33 @@ bool ParseConfigSingleWatchItem(NSString *name, std::string_view policy_version,
     }
   }
 
-  for (const PathAndTypePair &path_type_pair : std::get<PathAndTypeVec>(path_list)) {
-    data_policies.push_back(std::make_shared<DataWatchItemPolicy>(
-        NSStringToUTF8StringView(name), policy_version, path_type_pair.first, path_type_pair.second,
-        allow_read_access, audit_only, rule_type, enable_silent_mode, enable_silent_tty_mode,
-        NSStringToUTF8StringView(options[kWatchItemConfigKeyOptionsCustomMessage]),
-        options[kWatchItemConfigKeyOptionsEventDetailURL],
-        options[kWatchItemConfigKeyOptionsEventDetailText], std::get<PolicyProcessVec>(proc_list)));
+  switch (rule_type) {
+    case WatchItemRuleType::kPathsWithAllowedProcesses: [[fallthrough]];
+    case WatchItemRuleType::kPathsWithDeniedProcesses:
+      for (const PathAndTypePair &path_type_pair : std::get<PathAndTypeVec>(path_list)) {
+        data_policies.push_back(std::make_shared<DataWatchItemPolicy>(
+            NSStringToUTF8StringView(name), policy_version, path_type_pair.first,
+            path_type_pair.second, allow_read_access, audit_only, rule_type, enable_silent_mode,
+            enable_silent_tty_mode,
+            NSStringToUTF8StringView(options[kWatchItemConfigKeyOptionsCustomMessage]),
+            options[kWatchItemConfigKeyOptionsEventDetailURL],
+            options[kWatchItemConfigKeyOptionsEventDetailText],
+            std::get<PolicyProcessVec>(proc_list)));
+      }
+
+      break;
+
+    case WatchItemRuleType::kProcessesWithAllowedPaths: [[fallthrough]];
+    case WatchItemRuleType::kProcessesWithDeniedPaths:
+      proc_policies.insert(std::make_shared<ProcessWatchItemPolicy>(
+          NSStringToUTF8StringView(name), policy_version, std::get<PathAndTypeVec>(path_list),
+          allow_read_access, audit_only, rule_type, enable_silent_mode, enable_silent_tty_mode,
+          NSStringToUTF8StringView(options[kWatchItemConfigKeyOptionsCustomMessage]),
+          options[kWatchItemConfigKeyOptionsEventDetailURL],
+          options[kWatchItemConfigKeyOptionsEventDetailText],
+          std::get<PolicyProcessVec>(proc_list)));
+
```

Comment:
```suggestion
          std::get<PolicyProcessVec>(proc_list))
    );

```

Reply:
I'm on mobile so not sure the spacing is right but lining up the closing paren with the p or proc might be clearer

Reply:
I don't personally like that style, and it conflicts with the [google style guide](https://google.github.io/styleguide/cppguide.html#Function_Calls) for function calls. (clang-format also moves it back)

Reply:
Aight. I feel less strongly about this one, I just don't like seeing so many closing parens together cause it forces me to count